### PR TITLE
Add invitation to apply for alpha testing

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**

### CODE REVIEW

File: adoption.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    * using your own cloud accounts
+    * fully managed cloud hosting by DataJoint 
+ * Workflow execution  
+    * configuration and automation
+    * optional fully managed service by DataJoint Neuro
+ * Interfaces for data entry, export and publishing 
+
+During alpha and beta testing phases, these services may be subsidized by the grant funding for qualified research groups. 
+
+To select the support service, [apply here](https://elements.datajoint.io/apply)
```
Comment (guzman-raphael) [inline adoption.md:21]: We can just embed here the application to be an alpha tester/validation site. Have you guys determined what we need to collect when submitting?

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
You would also need to modify `docker-compose-build.yaml` so that the new file is accessible to the build utility. Also, `.mkdocs/mkdocs.yml` would need to be updated to reflect the new navigation as well. Have you tried running this locally with `docker-compose-dev.yaml`?

File: adoption.md (hunk 1 above)
Comment (dimitri-yatsenko) [reply adoption.md:21]: * Contact name and email
*  Institution and lab
*  How will you set up the MySQL server:
   (a) we will configure it ourselves
   (b) please set up a cloud server for our project for the testing period (10 sessions)

**Comment by guzman-raphael** [comment]
@dimitri-yatsenko I've issued a new PR #21 which will supersede this one and include the application to be a validation site.

**Pull request closed without merging**